ccc-js/src/main/javascript/package-res/lib/protovis.js

```diff
@@ -4521,14 +4520,17 @@ DateComponent.prototype.floor = function(d, options) {
 };
 
 DateComponent.prototype.floorMultiple = function(d, n, options) {
-  var first = this.first(d, options),
-      delta = this.get(d) - first; // base in units
-
+  var alignVal = pv.get(options, 'alignmentValue');
+  var alignRef = alignVal ? this.get(alignVal) : this.first(d, options);
```

Comment:
Please use `alignVal != null` to test for presence.

Reply:
👍